pkg/background/update_request_controller.go

```diff
@@ -171,6 +171,9 @@ func (c *controller) syncUpdateRequest(key string) error {
 	if err != nil {
 		return err
 	}
+	if ur.Spec.Context.AdmissionRequestInfo.AdmissionRequest != nil {
+		logger.Info("get ur", "object", string(ur.Spec.Context.AdmissionRequestInfo.AdmissionRequest.Object.Raw))
```

Comment:
Can we hide this default level log? Maybe set it to level 5?

Reply:
Yeah, I used it for debugging.